src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $white-color;
+  padding-top: 32px;
+
+  &__img {
+    display: flex;
```

Comment:
You have used `display: flex;` for the image, but there's no need to do this as there are no child elements to align. This line can be removed.